strapi/strapi
# fix(content-releases): fix padding bottom duplicated in the Heading of the Details page

### What does it do?

It removes the duplicated padding bottom we had in the Heading of the Details page

### Why is it needed?

The designs say 40px in total between the table and the heading description, you have 40px + the heading padding bottom, so 40+40=80px
<img width="1269" alt="Screenshot 2023-12-14 at 16 35 45" src="https://github.com/strapi/strapi/assets/2589748/f6740aab-8561-4e8f-8a54-2a5267cf36ea">


### How to test it?

- select a release from the content releases page and open its details page, and then check the padding bottom of the Header section
